layouts/index.html

```diff
@@ -13,8 +13,8 @@
     id="main">
   <div class="usa-grid section-bg-parent">
     <div class="usa-width-whole billboard  section-bg">
-      <h1 class="billboard-display_text">The Government Innovation Platform</h1>
-      <h2 class="billboard-second_text">A platform by government developers, for government developers.</h1>
+      <h1 class="billboard-display_text">A Platform as a Service for government teams</h1>
+      <h2 class="billboard-second_text">Focus on developing your applications. We manage the security, compliance, and maintenance of the virtual machines and servers.</h1>
```

Comment:
"maintenance of _virtual machines and servers_" seems redundant. Maybe "maintenance and hosting"?